### PR TITLE
Add Make.com extension

### PR DESCRIPTION
**Pull request by kkoscielniak**
Run Make.com Scenarios on demand.

This Extension lists the active Make.com Scenarios and allows you to trigger them directly from Raycast.

## Screenshot

make-com-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by kkoscielniak** [comment]
Hello @pernielsentikaer :) 

My wild guess would be that you didn't add the `scenarios:read`, `scenarios:write` and `scenarios:run` scopes for this particular API key. Could you try adding these scopes (you might need to create new API key)? If that's the case, I'll update the README. 

When developing this extension I've created a separate API key for testing that had **every** scope enabled (however this shouldn't be the case for the end users).

**Comment by pernielsentikaer** [comment]
I'm pretty sure I enabled everything possible when creating the API key but let me try again 🙂

**Comment by pernielsentikaer** [comment]
I still have the same issue @kkoscielniak - are you in our [Slack](https://www.raycast.com/community) community?

**Comment by kkoscielniak** [comment]
Hm, that’s strange. I will try to create a fresh account to see if I can reproduce the issue (I was using my private acc).

Regarding the community, I’ve logged in once. Once I am in front of my computer I’ll reach you on Slack :)

**Comment by pernielsentikaer** [comment]
Do that! Then we can try to debug with fresh credentials and you can also get access to the account I tested with 🙂
